posthog/hogql_queries/utils/test/test_query_date_range.py

```diff
@@ -45,16 +45,6 @@ def test_parsed_date_week(self):
         self.assertEqual(query_date_range.date_from(), parser.isoparse("2021-08-18 00:00:00Z"))
         self.assertEqual(query_date_range.date_to(), parser.isoparse("2021-08-25 23:59:59.999999Z"))
 
-    def test_is_hourly(self):
-        now = parser.isoparse("2021-08-25T00:00:00.000Z")
-        date_range = DateRange(date_from="-48h")
-
-        query_date_range = QueryDateRange(team=self.team, date_range=date_range, interval=IntervalType.day, now=now)
-        self.assertFalse(query_date_range.is_hourly)
-
-        query_date_range = QueryDateRange(team=self.team, date_range=date_range, interval=IntervalType.hour, now=now)
-        self.assertTrue(query_date_range.is_hourly)
```

Comment:
```    
@cached_property
def is_hourly(self) -> bool:
    return self.interval_name == "hour"
```
    
I deleted it because `is_hourly` was only used in like 3 places and just did this, and using `self.interval in ('hour', 'day')` checks is the colloquial way we're testing for this stuff all over the code base, so it was confusing that we used a function for it in a couple places